### PR TITLE
Add 3 TypedArray methods

### PR DESCRIPTION
**Pull request by NiedziolkaMichal**
Interactive examples for [TypedArray.forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/forEach), [TypedArray.reduceRight](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/reduceRight), [TypedArray.toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/toLocaleString).

### CODE REVIEW

File: live-examples/js-examples/typedarray/typedarray-reduceright.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const uint8 = new Uint8Array([10, 20, 30]);
+
+const result = uint8.reduceRight(
+  (accumulator, currentValue) => (`${accumulator}, ${currentValue}`)
+);
```
Comment (Josh-Cena) [inline live-examples/js-examples/typedarray/typedarray-reduceright.js]:
```suggestion
const result = uint8.reduceRight(
  (accumulator, currentValue) => `${accumulator}, ${currentValue}`,
);
```

File: live-examples/js-examples/typedarray/typedarray-foreach.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const uint8 = new Uint8Array([10, 20, 30]);
+
+uint8.forEach(element => console.log(element));
```
Comment (Josh-Cena) [inline live-examples/js-examples/typedarray/typedarray-foreach.js]:
```suggestion
uint8.forEach((element) => console.log(element));
```

File: live-examples/js-examples/typedarray/typedarray-tolocalestring.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const uint8 = new Uint32Array([500, 8123, 12]);
+
+console.log(uint8.toLocaleString());
+// expected output: "500,8123,12"
+
+console.log(uint8.toLocaleString('en-GB'));
+// expected output: "500,8,123,12"
+
+console.log(uint8.toLocaleString('de-DE', { style: 'currency', currency: 'EUR' }));
```
Comment (Josh-Cena) [inline live-examples/js-examples/typedarray/typedarray-tolocalestring.js:9]:
```suggestion
console.log(uint8.toLocaleString("en-GB"));
// expected output: "500,8,123,12"

console.log(uint8.toLocaleString("de-DE", { style: "currency", currency: "EUR" }));
```

**Review by Josh-Cena — COMMENTED** [review]
Some formatting and then this looks good

File: live-examples/js-examples/typedarray/typedarray-tolocalestring.js (hunk 3 above)
Comment (NiedziolkaMichal) [reply live-examples/js-examples/typedarray/typedarray-tolocalestring.js:9]: Currently, all examples use single quotes in code, while the double quote is used in `expected output`. I think it's better to stick with that way for now and let this example be formatted later when we add prettier.

Comment (Josh-Cena) [reply live-examples/js-examples/typedarray/typedarray-tolocalestring.js:9]: Ah okay. Thanks for the info

**Review by Josh-Cena — APPROVED** [review]
LGTM, thanks for all the work

**Comment by NiedziolkaMichal** [comment]
Thanks for reviews

**Pull request merged**